packages/marko-web-identity-x/browser/login.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div v-if="hasActiveUser">
+    <!-- @todo determine when new fields should display -->
+    <p>You are currently logged in as {{ activeUser.email }}.</p>
+    <a
+      :href="endpoints.logout"
+      class="btn btn-primary"
+      role="button"
+    >
+      {{ buttonLabels.logout }}
+    </a>
+  </div>
+  <div v-else-if="complete">
+    <h4>Almost Done!</h4>
+    <p>
+      We just sent an email to <em>{{ email }}</em> with your one-time login link.
+      To finish logging in, open the email message and click the link within.
+    </p>
+    <p>
+      Note: please check your spam/junk folders.
+      If you do not receive this email, your firewall or ISP has likely blocked it.
+      Please add noreply@identity-x.base-cms.io to your whitelist and try registering again.
```

Comment:
noreply@identity-x.io

Reply:
Yep. I was going to do all that separate of your other PR.